.gitignore

```diff
@@ -11,3 +11,6 @@ out/
 # gradle
 .gradle
 build/
+
```

Comment:
I use vim! And for other projects, I also .gitignore `*.swp`
You can also `set expandtab` in your `.vimrc` to insert spaces instead of tabs.
I can share my .vimrc if it'd help you.